docs/code_ref/lightcurve.rst

```diff
@@ -3,6 +3,11 @@
 SunPy lightcurve
 ================
 
+.. warning::
+
+   The lightcurve submodule has been replaced with the `sunpy.timeseries`
+   module. It will be removed in a future version.
```

Comment:
Fair enough. Didn't think it was clickable in the built version. Could just have been me.